tests/phpunit/tests/link/getEditTermLink.php

```diff
@@ -196,6 +196,25 @@ function ( $location, $term ) {
 
 		get_edit_term_link( $term, $taxonomy );
 	}
+	/**
+	 * This function checks if the return URL of `get_edit_term_link()` when called without taxonomy matches with the expected output or not.
+	 *
+	 * @ticket 61726
+	 *
+	 * @return void
+	 */
+	public function test_get_edit_term_link_without_taxonomy() {
+		$term1 = self::factory()->term->create(
+			array(
+				'taxonomy' => 'wptests_tax',
+				'name'     => 'bar',
+			)
+		);
```

Comment:
Creating a new term here makes the test slower and is unnecessary. Instead, you could reuse one of the terms created for this entire test class (see `wpSetUpBeforeClass()` method). There's also the `get_term()` helper method above that makes this a one-liner.
```suggestion
		$term1 = $this->get_term( 'wptests_tax', true );
```

Reply:
Fixed